docs/root/configuration/http/http_filters/jwt_authn_filter.rst

```diff
@@ -216,3 +207,29 @@ Above config uses more complex *group* requirements:
 
 * The first *rule* specifies *requires_any*; if any of **provider1** or **provider2** requirement is satisfied, the request is OK to proceed.
 * The second *rule* specifies *requires_all*; only if both **provider1** and **provider2** requirements are satisfied, the request is OK to proceed.
+
+Copy validated JWT claims to HTTP request headers example
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+:ref:`claim_to_headers <envoy_v3_api_field_extensions.filters.http.jwt_authn.v3.JwtProvider.claim_to_headers>` has two fields:
```

Comment:
line 214-218 seems odd at the top of section. Is this indented?  I would suggest move them below line 220

Reply:
Actually this paragraph can be re-worded as

The field claim_to_headers is a repeat of message JwtClaimToHeader(use :ref: for msg link,  it is a different format as field link).  The message JwtClaimToHeader has two fields:

...